src/components/Header/Header.jsx

```diff
@@ -1,6 +1,14 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import Navigation from '../Navigation/Navigation';
+import './Header.css';
 
-// Create a Header function returning the HTML of header block
+function Header() {
+  return (
+    <header className="header">
+      <h1 className="header__title">Site Name</h1>
+      <Navigation />
+    </header>
+  );
+}
```

Comment:
You also could rewrite your components 👇  it more commonly used.
```suggestion
const Header = () => (
      <header className="header">
        <h1 className="header__title">Site Name</h1>
        <Navigation />
      </header>
);
```